packages/maskbook/src/social-network-provider/twitter.com/ui/tasks.ts

```diff
@@ -75,6 +75,37 @@ const taskPasteIntoPostBox: SocialNetworkUI['taskPasteIntoPostBox'] = (text, opt
     worker(abortCtr).then(undefined, (e) => fail(e))
 }
 
+const taskImageShuffleUploadToPostBox: SocialNetworkUI['taskImageShuffleUploadToPostBox'] = async (
+    image: ArrayBuffer,
+    seed: string,
+    options: Parameters<SocialNetworkUI['taskUploadToPostBox']>[1],
+) => {
+    const {} = options
+    const shuffledImage = new Uint8Array(
+        decodeArrayBuffer(
+            await Services.ImageShuffle.shuffle(encodeArrayBuffer(image), {
+                seed,
+            }),
+        ),
+    )
+
+    // ! race conditions, thus sleep
+    await sleep(500)
```

Comment:
turns out this wasn't necessary in the first place. the original guy had this in here, I didn't question it. DOM appears not necessary since the paste function is called when the postboxdialog is ready